timesketch/frontend-ng/src/components/LeftPanel/AnalyzerResultTimeline.vue

```diff
@@ -289,12 +289,79 @@ limitations under the License.
           </tbody>
         </v-simple-table>
       </div>
-      <div v-else v-show="expanded" class="ml-3 pb-1 mr-2">
-        <v-icon>mdi-alert-octagon-outline</v-icon>
-        <span class="ml-1">
-          Showing multi analyzer results is not supported in the new UI yet. Please visit the old UI to see these
-          results.
-        </span>
+      <div v-else v-show="expanded"
+        :class="
+          $vuetify.theme.dark
+            ? expanded
+              ? 'dark-hover dark-bg'
+              : 'dark-hover'
+            : expanded
+            ? 'light-hover light-bg'
+            : 'light-hover'
+        ">
+        <!-- TODO: iterate on multianalyzer timeline results -->
+        <v-simple-table dense class="ml-2">
+          <tbody
+            :class="
+              $vuetify.theme.dark
+                ? expanded
+                  ? 'dark-hover dark-bg'
+                  : 'dark-hover'
+                : expanded
+                ? 'light-hover light-bg'
+                : 'light-hover'
+            "
+          >
+            <tr>
+              <td style="border: none">
```

Comment:
You could give the table a class "borderless" and then use 

```css
.borderless td {
  border: none;
}
```

Then it is easier to change, because you only have to change one part in the code.